website/docs/networking/vpc-lattice/latticeabtesting.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "A/B testing with VPC Lattice"
+sidebar_position: 20
+---
+
+In this section we will show how to use Amazon VPC Lattice to perform A/B testing gradually shifting traffic from the `checkout` service to a new version `checkout-v2`.
+
+# Set up Lattice Service Network
+
+1. Create the Kubernetes Gateway `eks-workshop-gw`:
+   ```bash
+   $ kubectl apply -f workspace/modules/networking/vpc-lattice/controller/eks-workshop-gw.yaml
+   ```
+1. Verify that `eks-workshop-gw` gateway is created:
+   ```bash
+   $ kubectl get gateway  
+   ```
+   ```
+   $ NAME              CLASS         ADDRESS   READY   AGE
+   $ eks-workshop-gw   aws-lattice                     12min
+   ```
+
+1. Once the gateway is created, find the VPC Lattice service network. Wait until the status is `Reconciled` (this could take about five minutes).
```

Comment:
Can we use a `kubectl wait` here? Or a custom command like here: https://github.com/aws-samples/eks-workshop-v2/blob/main/environment/bin/wait-for-lb (this is used in the Fundamentals load balancer module)